modules/apps/frontend-data-set/frontend-data-set-web/src/main/resources/META-INF/resources/cell_renderers/DateRenderer.js

```diff
@@ -27,12 +27,13 @@ function DateRenderer({options, value}) {
 		}
 	}
 
-	const locale = themeDisplay.getBCP47LanguageId();
+	const locale = Liferay.ThemeDisplay.getBCP47LanguageId();
 
 	const dateOptions = {
 		day: options?.format?.day || 'numeric',
 		month: options?.format?.month || 'short',
-		timeZone: options?.format?.timeZone || 'UTC',
+		timeZone:
+			options?.format?.timeZone || Liferay.ThemeDisplay.getTimeZone(),
```

Comment:
If a customer wants date to be displayed unchanged, they need to know that they must set `options.format.timeZone` to `UTC`. This seems pretty unintuitive to understand. Is it acceptable? 

This may cause regressions in existing usages of FDS that have dates. Do we maybe want this behavior as an opt-in?

Reply:
According to product, it looks like we want to convert by default.

I kept the option to not convert to user timezone by forcing DXP apps to explicitly pass `options.format.timeZone` value, but that as you say, makes DXP apps to choose a target timezone. This makes it hard to tell FDS to "use date with no conversion"

I think we can improve this in a followup, so that apps have a chance to say "use dates as they come" rather than just choosing an specific timezone. Makes sense @herrardo ?

Reply:
If the new behavior is not an opt-in, this will be a breaking change. Anybody using FDS with a date column will be affected. Their current solution, for better or worse, shows unchanged date, and that is expected behavior. After this change, the time we display will suddenly shift. If we are fine with this, we should document workaround, either setting of `options.format.timeZone` to `UTC` or the followup improvement.